.github/workflows/test.yaml

```diff
@@ -0,0 +1,54 @@
+name: Test Volatility3
+on: [push]
```

Comment:
Push is good for the test, but it would be great if it is possible to pre-test during the Pull Request stage. Most contributors will test it by turning it around themselves, but if we can test it in a common environment, we will be able to reduce errors. Of course, this may vary depending on the available Github Actions in Organization.

Reply:
@npetroni , shall I also create a workflow for testing on PRs? Or do you think that we should just perform *this* test on PRs as well as pushes?

Reply:
Changed in https://github.com/volatilityfoundation/volatility3/pull/789/commits/f96f004b13c15b3c0334f7ca96b2c0459a8f9cf1